tutorials/04-switching-physics-engines.md

```diff
@@ -0,0 +1,99 @@
+\page switchphysicsengines Physics engines
+
+This tutorial describes how to switch between physics engines in Ignition Physics when using Ignition Gazebo.
+Ignition Physics enables Ignition Gazebo to choose flexibly what physics engine to use at runtime.
+By default, Ignition Gazebo uses the [DART](https://dartsim.github.io/) physics engine.
+
+Downstream developers may also integrate other physics engines by creating new Ignition Physics engine plugins.
+See [Ignition Physics](https://ignitionrobotics.org/api/physics/3.0/tutorials.html)'s tutorials to learn how to integrate a new engine.
+
+## How Ignition Gazebo finds physics engines
+
+Ignition Gazebo automatically looks for all physics engine plugins that are installed with Ignition Physics.
+At the time of writing, there are two physics engines available (more detail in \ref physicsplugin "Physics plugin tutorial"):
+- **DART**: `ignition-physics-dartsim-plugin`.
+- **TPE**: `ignition-physics-tpe-plugin`.
+
+If you've created a custom engine plugin, you can tell Gazebo where to find it by setting the `IGN_GAZEBO_PHYSICS_ENGINE_PATH` environment variable to the directory where the plugin's shared library can be found.
```

Comment:
`... you can tell Gazebo..` -> `... you can tell Ignition Gazebo...`

just to be a little more specific since there is also Gazebo Classic.